features/tournament/optional_place_3_match.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Tournament optional place 3 match
+  As a user
+  In order to easily create a tournament
+  I want to make the place 3 match optional
+
+  Scenario: Create match without place 3 match
+    Given a user
+    And the user is logged in
+    When he visits the create event path
+    And he clicks 'New tournament'
+    And he fills in valid tournament data
+    And he unchecks the checkbox 'Spiel um Platz 3'
+    And he clicks 'Create'
+
+    Given 8 teams join the tournament
+    And the schedule page is visited
+
+    Then the text Spiel um Platz 3 is not there
```

Comment:
Check the data instead of the page.

Reply:
The data should be tested as well, which it is not, when this step fails, either the data is wrong or the view is incorrect, which is it?